src/components/Cart.js

```diff
@@ -0,0 +1,17 @@
+import { Component } from "react";
+import formatPrice from "../helpers/formatPrice";
+
+class Cart extends Component {
+    render(){
+        let { name, price } = this.props.item;
+        return (
+            <div>
+                <ul>
+                    <li>{name}: {formatPrice(price)}</li>
+                </ul>
+            </div>
+        )
```

Comment:
```suggestion
        return (
                      <h2>Cart</h2>
          <ul>
            {this.props.cartArr}
          </ul>
          <h3>Subtotal: {formatPrice(this.props.subtotal)}</h3>
          <h3>Tax: {formatPrice(this.props.subtotal * .05)}</h3>
          <h3>Total: {formatPrice(this.props.subtotal * 1.05)}</h3>
        )
```
Rememer, `this.props.cartArr` is an array of `<li>`'s, so it should be encased in a `<ul>`